Makefile

```diff
@@ -141,6 +141,11 @@ test-js-build: node-version-check
 
 test-js: node-version-check
 	@echo "--> Running JavaScript tests"
+	@$(YARN) run test
+	@echo ""
+
+test-js-ci: node-version-check
+	@echo "--> Running CI JavaScript tests"
 	@$(YARN) run test-ci
```

Comment:
Interesting that this was `test-ci` before, not `test`. @billyvg is `yarn run test` the correct way to run all js tests locally?

Reply:
Yeah. Run is unnecessary but this is fine